.github/renovate.json

```diff
@@ -205,6 +205,11 @@
       "matchPackagePatterns": ["^tracing[-_]?"],
       "excludePackageNames": ["tracing-opentelemetry"],
       "groupName": "`tracing` crates"
+    },
```

Comment:
Specific reason why anyhow is being singled out into it's own group?

Reply:
(Non blocking just curious)

Reply:
Renovate tries updating `anyhow` all the time but we only have a patch version specified here for nightly compatibility. After all this is a library and `anyhow` is an optional feature and we should not put stricter requirements on it than needed.